x-pack/platform/plugins/shared/streams/server/lib/streams/component_templates/logs_layer.ts

```diff
@@ -19,3 +29,186 @@ export const logsSettings: IndicesIndexSettings = {
     },
   },
 };
+
+export const baseFields: FieldDefinition = {
+  '@timestamp': {
+    type: 'date',
+  },
+  'stream.name': {
+    type: 'system',
+  },
+  'scope.dropped_attributes_count': {
```

Comment:
The [otel spec demands](https://opentelemetry.io/docs/specs/otel/common/mapping-to-non-otlp/#instrumentationscope) these fields are `otel.scope` but we use `scope`. Same for `otel.status_code` and `otel.status_description`.

This has never been a problem but technically we don't follow the spec.

Reply:
Good point, these are aligned with https://github.com/elastic/elasticsearch/blob/314dc1a20e4ac7cd78c4fe9e585975dd265194b3/x-pack/plugin/otel-data/src/main/resources/component-templates/otel%40mappings.yaml#L30 , which should be what our exporter is generating.

@felixbarny is there a special reason we didn't follow the naming of the spec in this particular place?